harmonica/synthetic/surveys.py

```diff
@@ -0,0 +1,145 @@
+"""
+Create synthetic surveys for gravity and magnetic observations
+"""
+import pyproj
+from verde import get_region, inside
+from verde.coordinates import check_region
+
+from ..datasets import fetch_britain_magnetic, fetch_south_africa_gravity
+
+
+def airborne_survey(region, cut_region=(-5.0, -4.0, 56.0, 56.5)):
```

Comment:
We might want to set default values for the `region`. For example, keeping it the same as the cut region by default.

Reply:
```suggestion
def airborne_survey(region, subsection=(-5.0, -4.0, 56.0, 56.5)):
```
Maybe? `cut_region` sounds a bit strange to me.

Reply:
I hate when I name variables while I'm drafting the new functions and after a few commits I cannot thing a better name for them because I've already got used to the ugly one. `subsection` is way better.

Reply:
What about `subregion`? Or `dataset_subregion`?

Reply:
`dataset_region` to be more explicit? 

> There are 2 hard problems in computer science: cache invalidation, naming things, and off-by-1 errors. -- [Leon Bambrick](https://twitter.com/secretGeek/status/7269997868)

Reply:
Yes, it sounds a little better than subsection.